.env.alfajores

```diff
@@ -50,10 +50,10 @@ AZURE_ORACLE_CENTRALUS_FULL_NODES_DISK_SIZE=30
 
 # Temporarily point to celo-org repository to consume patched image.
 GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-org/geth"
-GETH_NODE_DOCKER_IMAGE_TAG="1.1.0"
+GETH_NODE_DOCKER_IMAGE_TAG="1.2.2"
 
 GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
```

Comment:
We've switched the the geth node to `celo-org`, we could probably do that here as well

Reply:
@trianglesphere you mean for the bootnode image?

Reply:
Yea, that was what I was referring to. Given that it needs master, and not 1.2.2, I'm fine with leaving this as is for now.